### PR TITLE
feat: Removed hm-upnp container. Its task is now carried by hm-config

### PR DESCRIPTION
**Pull request by MuratUrsavas**
(NebraLtd/hm-config#164)

The overhead of hm-upnp became unnecessary and we needed to decommission it from the product. The task of the upnp container is now carried by gateway-config.

**How**
- Removed UPNP parameter from settings.ini 
- Removed upnp container section from the container configuration template

**References**
NebraLtd/hm-config#164

**Checklist**

- [x] Tests added
- [x] Cleaned up commit history (rebase!)
- [x] Documentation added
- [x] Thought about variable and method names

### CODE REVIEW

File: settings.ini
Comment (shawaj) [inline settings.ini:7]: Need to bump config version too before merging

Comment (MuratUrsavas) [reply settings.ini:7]: Done. But it is the one from my branch. Cool?

Comment (MuratUrsavas) [reply settings.ini:7]: Got the answer, will wait for merge.

Comment (MuratUrsavas) [reply settings.ini:7]: Done.

**Review by kashifpk — APPROVED** [review]

**Comment by vpetersson** [comment]
Woop woop!

**Comment by shawaj** [comment]
@MuratUrsavas you should bump the firmware version number in settings.ini for every PR.

**Comment by MuratUrsavas** [comment]
Yep, my bad. Fixed all other things and missed that.

**Comment by shawaj** [comment]
> Yep, my bad. Fixed all other things and missed that.

No worries, sorted it now 👌

**Pull request merged**
